### DISCUSSION TITLE
关于委托过滤器中日常石油委托的一个问题

### DISCUSSION THREAD

**Discussion by Bread200** [LmeSzinc/AzurLaneAutoScript#2936] [Q&A]
日常和夜间石油委托同样的时间会有三种类型，
我根据性价比做了个简单表格（包含日常石油和紧急委托中钻头和箱子委托，红尖尖必做没列入其中），
可以看出三种类型的差别中①型收益不高但③型收益较高，
Q1：请问委托过滤器中可以作区分吗？
Q2：如果不行的话，alas不做区分的原因是每日委托石油收益差别不大没有必要还是有其他原因呢？
![image](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/137496220/0e864121-7000-4d8e-a2f3-2c2f113fbe50)

**Comment by LmeSzinc** [comment]
https://github.com/LmeSzinc/AzurLaneAutoScript/wiki/filter_string_cn#%E5%A7%94%E6%89%98%E8%BF%87%E6%BB%A4%E5%99%A8

**Reply by Bread200** [reply]
我已经仔细读过这个过滤器了，我好像还是没有看到这和我提的问题有什么关系，
举个例子，ExtraOil-1有三个类型
![image](https://github.com/LmeSzinc/AzurLaneAutoScript/assets/137496220/403ff37b-4cb8-478a-9715-3a9b954ff49c)
这三个里面小型石油①和小型石油③能在过滤器中区别出来吗
